src/base/bittorrent/session.h

```diff
@@ -291,8 +291,8 @@ namespace BitTorrent
         bool isBandwidthSchedulerEnabled() const;
         void setBandwidthSchedulerEnabled(bool enabled);
 
-        uint saveResumeDataInterval() const;
-        void setSaveResumeDataInterval(uint value);
+        int saveResumeDataInterval() const;
```

Comment:
Changing types like this one should have its own commit.

Reply:
fixed